stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -93,10 +94,10 @@ class Process():
     def is_alive(self) -> bool: ...
     def join(self, timeout: Optional[float] = ...) -> None: ...
 
-class Queue():
+class Queue(queue.Queue):
```

Comment:
I'm not that familiar with the mypy types yet, so I'm not sure if I'm doing this correctly, but changing it to a `Generic[_T]` like queue.Queue gives me `error: Need type annotation for variable` when I do `q = mp.Queue()`